relayer/chains/substrate/event_parser.go

```diff
@@ -312,22 +365,33 @@ func (con *connectionInfo) MarshalLogObject(enc zapcore.ObjectEncoder) error {
 func (con *connectionInfo) parseAttrs(log *zap.Logger, attributes interface{}) {
 	attrs := attributes.(ibcEventQueryItem)
 
+	var err error
+
+	var height exported.Height
+	if height, err = parseHeight(attrs["height"]); err != nil {
+		log.Error("error parsing connection height: ",
```

Comment:
Logs should begin with a capital letter and not end in punctuation.

See https://github.com/cosmos/relayer/blob/main/docs/logging.md